### DISCUSSION TITLE
Introduction and Request for Guidance on Contributing to RUXAILAB 🚀

### DISCUSSION THREAD

**Discussion by sumana-2705** [ruxailab/RUXAILAB#539] [General]
Hello👋
My name is Sumana Sree. I'm a Master's student at IIT (BHU), specializing in Machine Learning and AI. I enjoy contributing to open-source projects and have recently completed Hacktoberfest with more than 4 merged PRs. I am new to this organisation. I want to contribute to RUXAILAB, I have experience in OpenCV, Deep Learning and NLP. Can anyone help to find the issues related to these topics in the repositories. 

Thank you!!!

**Comment by marcgc21** [comment]
Hi @sumana-2705  we are closing the creation of threads inside Discussions because it is hard to maintain for the organization.

 We prefer to work on Discord channels. Feel free to reach me there and talk about what experience!
